### DISCUSSION TITLE
cypress-on-rails gem Error: File unchanged! The supplied flag value not found!  cypress/support/index.js

### DISCUSSION THREAD

**Discussion by TahirMehmood8082** [shakacode/cypress-playwright-on-rails#143] [General]
while running this cmd in rails terminal
**rails g cypress_on_rails:install**
**i got this error**
myProject/**playwright**/support/index.js does not appear to exist
it unable to find **index.js** and **playwright** folder never exist????

cypress folder created by this command itself does not have playwright folder, but have index.js in myProject/support/index.js
Any solution???

**here is log**
create  cypress/support/index.js
      create  cypress/support/commands.js
      create  cypress.config.js
   identical  config/initializers/cypress_on_rails.rb
   identical  e2e/e2e_helper.rb
       exist  e2e/app_commands
   identical  e2e/app_commands/activerecord_fixtures.rb
   identical  e2e/app_commands/clean.rb
   identical  e2e/app_commands/eval.rb
   identical  e2e/app_commands/factory_bot.rb
   identical  e2e/app_commands/log_fail.rb
   identical  e2e/app_commands/scenarios/basic.rb
      create  cypress/support/on-rails.js
      create  cypress/e2e/rails_examples
      create  cypress/e2e/rails_examples/advance_factory_bot.cy.js
      create  cypress/e2e/rails_examples/other.cy.js
      create  cypress/e2e/rails_examples/using_factory_bot.cy.js
      create  cypress/e2e/rails_examples/using_fixtures.cy.js
      create  cypress/e2e/rails_examples/using_scenarios.cy.js
      create  cypress/e2e/rails_examples/using_vcr.cy.js
**File unchanged! The supplied flag value not found!  cypress/support/index.js
The file /media/infiniti/projects/Office-Work/Rails/runners_connect/playwright/support/index.js does not appear to exist**

**Comment by Judahmeek** [comment]
I responded to the issue you created regarding this error.
